lighthouse/types.py

```diff
@@ -12,28 +12,40 @@
 EventMessage = Dict[str, Union[str, Event]]
 
 
+class EventPropertyProtocol(Protocol):
+    @property
+    def _params(self) -> Dict[str, str]:
```

Comment:
I think `@property` infers public accessible properties but this has an underscore in the front making it conflicting?

Reply:
Yes, also from what I read is not right to use private methods not belonging to the mixin, so I'll replace it with a public method.

Reply:
done